CAIPs/caip-25.md

```diff
@@ -25,11 +25,37 @@ The motivation comes from the lack of standardization across blockchains to expo
 
 ## Specification
 
-The session is proposed by a caller and the response by the respondent is used as the baseline for an ongoing session that both parties will persist.
-The properties and authorization scopes that make up the session are expected to be persisted and tracked over time by both parties in a discrete data store, identified by an entropic [identifier][CAIP-171] assigned in the initial response.
-This object gets updated, extended, closed, etc. by successive calls and notifications, each tagged by this identifier.
+### Language
 
-If a respondent (e.g. a wallet) needs to initiate a new session, whether due to user input, security policy, or session expiry reasons, it can simply generate a new session identifier to signal this notification to the calling provider; if a caller needs to initiate a new session, it can do so by sending a new request without a `sessionIdentifier`. In such cases, a respondent (e.g. wallet) may choose to explicitly close all sessions upon generation of a new one from the same origin or identity, or leave it to time-out; maintaining concurrent sessions is discouraged (see Security Considerations).
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" written in
+uppercase in this document are to be interpreted as described in [RFC
+2119][]
+
+### Definition
```

Comment:
```suggestion
### Definition
```
```suggestion
### Definition

#### Session Lifecycle
```

Reply:
I think this section has gotten long enough to warrant some subheadings for future anchor-links to point to :D